### DISCUSSION TITLE
Kanallisten update

### DISCUSSION THREAD

**Discussion by polo87** [PredatH0r/ChanSort#377] [General]
die Kanalliste kann nicht Sortiert werden, Es bleiben die Original# immer erhalten

**Comment by PredatH0r** [comment]
Bitte Marke, Modellnummer, Originalliste (.bak) und geänderte Liste bekannt geben, sonst kann ich nicht helfen.

**Reply by polo87** [reply]
Sorry mein Fehler,
Panasonic TV, Liste wird eingelesen und von mir umsortiert, soweit  alles ok, allerdings fehlt mir der Button um die PR# Aufsteigend neu vergeben zu können in der Version [ChanSort 2023-10-28], oder ich sehe den Wald vor Bäumen nicht.

Danke
Stefan

**Comment by PredatH0r** [comment]
Es hängt davon ab, ob man beim Öffnen der Liste "Bestehende Liste bearbeiten" oder "Neu zusammenstellen" auswählt. 
Je nachdem öffnet sich die Ansicht mit einer Liste oder 2 Listen und die Liste hat entweder die alten oder keine Programmnummern voreingestellt. Im Menü "Einstellungen" kann man das auch nachträglich die 2-Tabellen-Ansicht ein/ausschalten.

Stellt man eine neue Liste zusammen, ist diese zunächst leer und man doppelklickt in der rechten Liste dann einfach der Reihe nach die Sender an, die man haben möchte.
Beim Ändern der bestehenden List kann man entweder die Programmnummer direkt überschreiben, oder mit den Buttons rauf/runter (grüner Pfeil) ändern.

Hat die bestehende Liste Lücken, die man loswerden will, markiert man die gewünschten Sender (Strg+A für alle, klick und Umschalt+Klick für eine Bereich, Strg+Klick für zusätzliche Auswahl, ...) und wählt dann im Menü "Bearbeiten / Sender neu nummerieren".
![grafik](https://github.com/PredatH0r/ChanSort/assets/6238104/df6ad890-a889-494c-8f08-1ffa8a993897)

**Reply by polo87** [reply]
Vielen Dank für die Informationen, werde ich gleich testen.
Schönen Abend noch
Gruß
Stefan
